src/components/ContextMenu/ContextMenu.less

```diff
@@ -0,0 +1,7 @@
+.lucid-ContextMenu {
+	display: inline-block;
+}
+
+.lucid-ContextMenu-FlyOut {
```

Comment:
i thought it should be consistent with both the child component name and the props name